slick/slick.js

```diff
@@ -1358,23 +1358,26 @@ Issues: http://github.com/kenwheeler/slick/issues
 
 	Slick.prototype.initDotEvents = function() {
 
-		var _ = this;
-
-		if (_.options.dots === true && _.slideCount > _.options.slidesToShow) {
-			$('li', _.$dots).on('click.slick', {
-				message: 'index'
-			}, _.changeSlide);
-		}
-
-		if ( _.options.dots === true && _.options.pauseOnDotsHover === true ) {
-
-			$('li', _.$dots)
-				.on('mouseenter.slick', _.interrupt.bind(_, true))
-				.on('mouseleave.slick', _.interrupt.bind(_, false));
-
-		}
-
-	};
+        var _ = this;
```

Comment:
there's indentation problems. 

please include the rules of the .editorconfig in your project before editing.